src/FFMpeg/AdvancedOutputMapping.php

```diff
@@ -59,7 +59,7 @@ public function apply(AdvancedMedia $advancedMedia): void
             $parameters = $this->format->getAdditionalParameters() ?: [];
 
             if (!in_array('-b:v', $parameters) && $this->format->getKiloBitrate() !== 0) {
-                $parameters = ['-b:v', $this->format->getKiloBitrate() . 'k'] + $parameters;
```

Comment:
This feels like parameters was broken before. But if parameters ever have keys it will cause unexpected behaviors.

```
var_dump(['a', 'b'] + ['c']);
```

> ["a", "b"] 

```
var_dump(['a'=>1, 'b'=>2] + ['c' =>3]);
```

> ["a" => 1, "b" => 2, "c" => 3]